src/videojs-http-streaming.js

```diff
@@ -285,12 +285,9 @@ class HlsHandler extends Component {
     this.ignoreNextSeekingEvent_ = false;
     this.setOptions_();
 
-    // overriding native HLS only works if audio tracks have been emulated
-    // error early if we're misconfigured:
-    if (this.options_.overrideNative &&
-        (tech.featuresNativeVideoTracks || tech.featuresNativeAudioTracks)) {
-      throw new Error('Overriding native HLS requires emulated tracks. ' +
-        'See https://git.io/vMpjB');
+    if (this.options_.overrideNative) {
```

Comment:
We'll need to fall back to the old behavior if the APIs aren't available (older video.js 6)